server/src/test/java/org/elasticsearch/common/RoundingTests.java

```diff
@@ -469,6 +469,20 @@ public void testRandomTimeIntervalRounding() {
         }
     }
 
+    public void testFoo() {
+        Rounding rounding = new Rounding.TimeIntervalRounding(960000, ZoneId.of("Europe/Minsk"));
+        long rounded = rounding.prepareForUnknown().round(877824908400L);
+        long next = rounding.prepareForUnknown().nextRoundingValue(rounded);
+        assertThat(next, greaterThan(rounded));
+    }
+
+    public void testBar() {
+        Rounding rounding = new Rounding.TimeIntervalRounding(480000, ZoneId.of("Portugal"));
+        long rounded = rounding.prepareJavaTime().round(972780720000L);
+        long next = rounding.prepareJavaTime().nextRoundingValue(rounded);
+        assertThat(next, greaterThan(rounded));
+    }
```

Comment:
I'm not sure I need to keep these. They failed on some of my other attempts at implementing this method.

Reply:
I'd keep them in, personally.  Could just put a note that those specific values caused problems for some implementations, so we're keeping them around for regression testing, or something like that.